library/src/main/java/com/cube/storm/ui/lib/spec/ChevronSpec.java

```diff
@@ -0,0 +1,86 @@
+package com.cube.storm.ui.lib.spec;
+
+import android.text.TextUtils;
+
+import com.cube.storm.ui.model.list.StandardListItem;
+import com.cube.storm.ui.model.property.DestinationLinkProperty;
+import com.cube.storm.ui.model.property.LinkProperty;
+import com.cube.storm.ui.model.property.ShareLinkProperty;
+import com.cube.storm.ui.view.holder.list.StandardListItemViewHolder;
+
+/**
+ * Specification abstract class used to determine whether a chevron should be shown on each {@link StandardListItemViewHolder}.
+ * Use this abstract class along with {@link com.cube.storm.UiSettings#setChevronSpec(ChevronSpec)} to change the default spec.
+ * Currently defaults to {@link ChevronSpec#noChevronSpec()}.
+ *
+ * @author JR Mitchell
+ * @project LightningUI
+ */
+public abstract class ChevronSpec
```

Comment:
Guessing the preference of `abstract` to an `interface` is for the static methods (since there is only 1 instance method which is abstract)?
Just wondering if you can use nullability of a `ChevronSpec` to check if a cell should have chevrons.
But more than happy to leave, I'm just getting your thoughts really.

Reply:
You're right, I chose to make it `abstract` rather than `interface` for the static methods... although I could potentially make it an `interface` and pull the default implementations out as class implementations rather than methods. Do you think this would be a better approach?

As for the nullability, I understand the approach but I think I'd rather keep it non-null, as it's less obvious (imo) what a `null` `ChevronSpec` means than what a `ChevronSpec` with `shouldChevronShow() = false` means